api/app.py

```diff
@@ -2,13 +2,13 @@
 import yaml
 from flask import Flask, jsonify
 from flasgger import Swagger
-from api.prints.swagger.swag import SwaggerDoc
-from api.db import db
-from api.elastic import elastic
-from api.prints import base, search, uuid
+from prints.swagger.swag import SwaggerDoc
```

Comment:
I think the py3 way to do this is 
`from .prints.swagger.swag import SwaggerDoc`

Reply:
etc. etc.